imap/user.go

```diff
@@ -176,8 +176,13 @@ func (u *user) initMailboxes() error {
 			if _, ok := u.mailboxes[label.ID]; ok {
 				continue
 			}
-
-			u.mailboxes[label.ID], err = newMailbox(label.Name, label.ID, nil, u)
+			var displayName string
+			if len(label.Path) > 0 {
+				displayName = strings.ReplaceAll(label.Path, "/", ".")
```

Comment:
`.` might be used elsewhere in the name. Instead of replacing `/` with `.`, could we define the IMAP separator to be `/`?

Reply:
Yeah there is definitely a better way to do it so the folder hierarchy is properly exposed via IMAP. I didn't look into what would be required in the IMAP code to properly parse and expose the folder names.

Reply:
I don't think anything special should be required, apart from using the right separator.

Reply:
Ok I'll do some testing and try to come up with a better PR.